Cargo.toml

```diff
@@ -16,25 +16,26 @@ atlas = []
 render = []
 
 [dependencies]
-bevy = { version = "0.11", default-features = false, features = [
+bevy = { version = "0.12", default-features = false, features = [
     "bevy_core_pipeline",
     "bevy_render",
     "bevy_asset",
     "bevy_sprite",
+    "multi-threaded"
```

Comment:
Is this also needed here? I think this might only be necessary in `dev-dependencies`.

Reply:
Ah, it is not, good catch!